src/index.js

```diff
@@ -1,23 +1,34 @@
-'use strict'
+// @flow
+"use strict"
 
-const multihash = require('multihashes')
-const blake = require('./blake')
-const sha3 = require('./sha3')
-const crypto = require('webcrypto')
+import * as multihash from "multihashes"
+import type { Multihash } from "multihashes"
+import type { Name, Code } from "multihashes/lib/constants"
+import type { HashTable, Hash } from "./types"
+import * as blake from "./blake"
+import * as sha3 from "./sha3"
+import * as crypto from "webcrypto"
 
-const mh = module.exports = Multihashing
+const mh = (module.exports = Multihashing)
```

Comment:
I would suggest `export default Multihashing`

Reply:
@garrensmith In fact [flow-node](https://github.com/Gozala/babel-preset-flow-node) babel preset takes care of this as it includes [add-module-exports](https://github.com/59naga/babel-plugin-add-module-exports) plugin only thing to watch out for is that you can either single `default export` or multiple non default exports (as there is no way around it with common-js style).